pkgs/games/mar1d/default.nix

```diff
@@ -62,9 +39,9 @@ stdenv.mkDerivation rec {
       original, however, the game still takes place in a two dimensional world.
       You must view the world as mario does, as a one dimensional line.
     '';
-    homepage = "https://github.com/olynch/fp_mario";
+    homepage = "https://mar1d.com";
     license = licenses.agpl3;
     maintainers = with maintainers; [ taeer ];
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.unix;
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
Oh, unix includes linux?

Reply:
Just tracked down where that's defined. I haven't actually tested this on any of the other platforms under "unix" (BSD, SunOS, etc). Should I change this to `platforms.linux ++ platforms.darwin`?

Reply:
> Oh, unix includes linux?

yes.

> I haven't actually tested this on any of the other platforms under "unix" (BSD, SunOS, etc).

Thats not expected right now because bsd is almost completely broken right now anyway.

Reply:
so "unix" essentially just means `linux ++ darwin` right now? I'm fine with it being either `unix` or `linux ++ darwin`, i just don't want to misrepresent things.